astropy/coordinates/sky_coordinate_parsers.py

```diff
@@ -659,41 +653,29 @@ def _parse_ra_dec(coord_str):
     ----------
     coord_str : str
         Coordinate string to parse.
+    is_ra_dec_representation : bool
+        Whether the coordinate should be assumed to represent a right
+        ascension and declination.
 
     Returns
     -------
-    coord : str or list of str
+    coord : list of str
```

Comment:
the previous implementation actually returned a tuple, and since the number of elements is invariant and small, I think returning a `tuple[str, str]` is more idiomatic and even more restrictive (more is better)

Reply:
The docstring and a comment close to where this function is called explicitly mentioned lists, and the last `else` statement of the old code actually did cause a `list` to be returned, so I thought that making the function always return a `list` is less disruptive. But you are right in saying that `tuple[str, str]` is a better return type, so I've adopted your suggestion.